doc/release-notes-27114.md

```diff
@@ -0,0 +1,2 @@
+- Additional flags "in" and "out" have been added to `-whitelist` to control whether
+  permissions apply to incoming connections and/or manual (default: incoming only).
```

Comment:
Kinda funny that in/out maps to in/manual ... perhaps `"manual"` should've been the permission flag?

Reply:
I'll leave it as is for while since it is specified that it applies only for manual connections.